seed.js

```diff
@@ -0,0 +1,214 @@
+    name: "Surette Provision Saison",
+    description: `In a particular region of southern Belgium, the reserve or provision saison was called surette, meaning tart. Using time-honored techniques, our saisons employ old-world methods to restore complexity and the classical rustic character lost in modern day saison. Brewed with local Colorado floor malted barley, traditional harvest grains, and noble hops, Surette embodies old-world saison brewing traditions. Each batch of Surette is an expression of the large oak foeders it matures in while taking on an aged vinous quality and a complex tartness. Showcasing citrusy ripe fruits, a light zesty spice, and earthy undertones, Surette’s character exemplifies our mixed culture of Brettanomyces and souring bacteria.`,
+    imageUrl: "/images/crooked_stave_surette.jpg",
+    price: 9.0
+  },
+  {
+    name: "Coffee Baltic Porter",
+    description: `Coffee Baltic Porter focuses on the art of coffee roasting, showcasing the uniqueness of this single ingredient. Through conditioning of of our Baltic porter with freshly roasted beans, we strive for a balanced coffee character, adding dark sugar, plum, and cocoa notes to the already robust porter base.`,
+    imageUrl: "/images/crooked_stave_coffee_baltic_porter.jpg",
+    price: 13.0
+  },
+  {
+    name: "Janiak Maniac",
+    description: `100% Bohemian Pilsner malt and new world hops. Unfiltered and unfazed.`,
+    imageUrl: "/images/kcbc_janiak_maniac.jpg",
+    price: 6.0
+  },
+  {
+    name: "Superhero Sidekicks",
+    description: `Bitten by the Hoptopus, our KCBC IPA is reborn! Ripe oranges, fresh pineapple and smooth pine. Mosaic, Idaho 7, Cascade & Centennial hops combine to deliver a powerful lupulin punch, backed by a balanced malt base. Soft haze that leans Northeast-ward, with a crisp, moderately-bitter finish for maximum drinkability.`,
+    imageUrl: "/images/kcbc_superhero_sidekicks.jpg",
+    price: 7.0
+  },
+  {
+    name: "Straight Jacket",
+    description: `Our Barleywine Ale features American and Belgian malts which provide the base of this extraordinary ale. After 12 months in bourbon barrels the award-winning Straight Jacket emerges with notes of stone fruit, toasted coconut, molasses, and vanilla coming in waves as you swirl the glass.`,
+    imageUrl: "/images/revolution_straight_jacket.jpg",
+    price: 10.0
+  },
+  {
+    name: `Ellie's Brown Ale`,
+    description: `This beautiful, deep russet brew has the sweet and somewhat nutty character of Adam Avery's late (1992-2002) Chocolate Lab, for which it is named.`,
+    imageUrl: "/images/avery_ellies.jpg",
+    price: 5.0
+  }
+];
+
+const seed = () => {
+  Promise.all(categories.map(category => Category.create(category)))
+    .then(seededCategories => {
+      Promise.all(suppliers.map(supplier => Supplier.create(supplier))).then(
+        seededSuppliers => {
+          Promise.all(products.map(product => Product.create(product))).then(
+            seededProducts => {
```

Comment:
Return an array of the set relationships.

Reply:
`Promise.all([
	seededProducts[1].setCategory(seededCategories[4]),
	// ...
])`